MarsCompetition/MarsCompetition/Pages/CertificationPage.cs

```diff
@@ -0,0 +1,157 @@
+﻿using DocumentFormat.OpenXml.Bibliography;
+using MarsCompetition.Utilities;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsCompetition.Pages
+{
+    public class CertificationPage : CommonDriver
+    {
+        private static IWebElement certificationsTab => driver.FindElement(By.XPath("//a[text() = 'Certifications']"));
+        private static IWebElement addNewButton => driver.FindElement(By.XPath("//div[@class='ui bottom attached tab segment tooltip-target active']//div[contains(@class,'ui teal button')][normalize-space()='Add New']"));
```

Comment:
I expected that you to break down your components in to smaller readable components, the framework to be created so that is easier to manage and maintain. You have not followed my task spec and have not thought about the layout! You have also just copied what you did for the competition task which out beats the purpose of learning advance concepts! Please have a look at the new task spec I have attached and make the changes so that it captures the advance concepts! I have put an example of how the framework should be laid out and how components should be broken down! I have also attached a github example if you would like to follow that!